learning/tour-of-beam/frontend/lib/config/theme/colors_provider.dart

```diff
@@ -67,20 +67,26 @@ class ThemeColors {
     );
   }
 
-  Color get divider => isDark ? kDarkGrey : kLightGrey;
+  Color get divider =>
+      isDark ? ProjectDarkThemeColors.grey : ProjectLightThemeColors.grey;
```

Comment:
It's better to name constants after their role and not their values, like `AppDarkThemeColors.divider`.

Reply:
One syntax may be even nicer:

`AppTheme.dark.divider`

To do this, you make a `class AppThemeColors` with `divider` and other fields. And then

```dart
abstract class AppTheme {
  static const dark = AppThemeColors(...);
}
```

Reply:
It's often problematic to name constants after their role, because one color or size can be used for unrelated purposes. It's also convenient to look for constants by value when working with mockups. I also get values by their role when it's useful, as in ThemeColors.divider.

Reply:
But then one day they split. We had it in Playground with `grey`. It was used for both dividers and button backgrounds and dropdown lists. Then we had to do an orange dropdown list and had troubles splitting them. So `divider` emerged.

`grey` is not used anywhere else yet. You can safely make it `TobDarkThemeColors.divider`.